runtime/kusama/src/lib.rs

```diff
@@ -1365,6 +1365,36 @@ impl frame_support::traits::OnRuntimeUpgrade for ParachainHostConfigurationMigra
 	}
 }
 
+pub struct GrandpaStoragePrefixMigration;
+impl frame_support::traits::OnRuntimeUpgrade for GrandpaStoragePrefixMigration {
+	fn on_runtime_upgrade() -> frame_support::weights::Weight {
+		use frame_support::traits::PalletInfo;
+		if let Some(name) = <Runtime as frame_system::Config>::PalletInfo::name::<Grandpa>() {
```

Comment:
I think we could use `expect` here as Grandpa is part of pallets in construct_runtime, thus it has been given a name.